optuna_integration/mlflow.py

```diff
@@ -207,8 +207,9 @@ def wrapper(trial: optuna.trial.Trial) -> float | Sequence[float]:
                     study = trial.study
                     self._initialize_experiment(study)
                     nested = self._mlflow_kwargs.get("nested")
+                    run_name = self._mlflow_kwargs.get("run_name") or str(trial.number)
```

Comment:
If the first value can be evaluated as `False` although I think this happens very unlikely, `str(trial.number)` is used, so how about using `str(trial.number)` as a default?

```suggestion
                    run_name = self._mlflow_kwargs.get("run_name", str(trial.number))
```

Reply:
sure, let's do that

Reply:
Ah, right. Thank you for your comment. Maybe we can update [`MLFlowCallback`](https://github.com/optuna/optuna-integration/blob/1c3fa01823c4e448d3773c63e7fd27e45e016b5c/optuna_integration/mlflow.py#L149)'s part too.

Reply:
Thanks. Sorry, the mlflow test failed due to my suggestion, let me check what was wrong...

Reply:
yep i updated the test because we were setting run_name at None in the test

Reply:
Awesome!